styles/new/typography.css

```diff
@@ -0,0 +1,34 @@
+h1 {
```

Comment:
I think you can simply apply font-semibold to h1,h2,...h6 altogether instead of repeating it on every h